[TZone] Add annotations to unannotated subclasses

msaboff opened the pull request:
#### a0f84835313943021fb8d6f68316bef611576e24
<pre>
[TZone] Add annotations to unannotated subclasses
<a href="https://bugs.webkit.org/show_bug.cgi?id=279191">https://bugs.webkit.org/show_bug.cgi?id=279191</a>
<a href="https://rdar.apple.com/135344436">rdar://135344436</a>

Reviewed by Mark Lam.

Added TZone allocation annotation to unannotated subclasses that inherit from annotated base classes.

* Source/WebCore/Modules/model-element/dummy/DummyModelPlayerProvider.cpp:
* Source/WebCore/Modules/model-element/dummy/DummyModelPlayerProvider.h:
* Source/WebCore/PAL/pal/text/TextCodecCJK.cpp:
* Source/WebCore/PAL/pal/text/TextCodecCJK.h:
* Source/WebCore/PAL/pal/text/TextCodecICU.cpp:
* Source/WebCore/PAL/pal/text/TextCodecICU.h:
* Source/WebCore/PAL/pal/text/TextCodecReplacement.cpp:
* Source/WebCore/PAL/pal/text/TextCodecReplacement.h:
* Source/WebCore/PAL/pal/text/TextCodecSingleByte.cpp:
* Source/WebCore/PAL/pal/text/TextCodecSingleByte.h:
* Source/WebCore/PAL/pal/text/TextCodecUTF16.cpp:
* Source/WebCore/PAL/pal/text/TextCodecUTF16.h:
* Source/WebCore/PAL/pal/text/TextCodecUTF8.cpp:
* Source/WebCore/PAL/pal/text/TextCodecUTF8.h:
* Source/WebCore/PAL/pal/text/TextCodecUserDefined.cpp:
* Source/WebCore/PAL/pal/text/TextCodecUserDefined.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmAesCbcCfbParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmAesCtrParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmAesGcmParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmAesKeyParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmEcKeyParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmEcdhKeyDeriveParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmEcdsaParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmHkdfParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmHmacKeyParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmPbkdf2Params.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmRsaHashedImportParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmRsaKeyGenParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmRsaOaepParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmRsaPssParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmX25519Params.h:
* Source/WebCore/dom/ActiveDOMObject.cpp:
* Source/WebCore/dom/EventLoop.cpp:
* Source/WebCore/editing/Editor.h:
* Source/WebCore/html/BaseCheckableInputType.cpp:
* Source/WebCore/html/BaseCheckableInputType.h:
* Source/WebCore/html/BaseClickableWithKeyInputType.cpp:
* Source/WebCore/html/BaseClickableWithKeyInputType.h:
* Source/WebCore/html/BaseDateAndTimeInputType.cpp:
* Source/WebCore/html/BaseDateAndTimeInputType.h:
* Source/WebCore/html/CheckboxInputType.cpp:
* Source/WebCore/html/CheckboxInputType.h:
* Source/WebCore/html/HTMLMediaElement.cpp:
* Source/WebCore/html/HiddenInputType.cpp:
* Source/WebCore/html/HiddenInputType.h:
* Source/WebCore/html/RadioInputType.cpp:
* Source/WebCore/html/RadioInputType.h:
* Source/WebCore/html/RangeInputType.cpp:
* Source/WebCore/html/RangeInputType.h:
* Source/WebCore/html/TextFieldInputType.cpp:
* Source/WebCore/html/TextFieldInputType.h:
* Source/WebCore/loader/EmptyClients.cpp:
* Source/WebCore/loader/MediaResourceLoader.cpp:
* Source/WebCore/loader/MediaResourceLoader.h:
* Source/WebCore/page/Chrome.cpp:
* Source/WebCore/page/Chrome.h:
* Source/WebCore/page/DummySpeechRecognitionProvider.h:
* Source/WebCore/page/csp/ContentSecurityPolicyMediaListDirective.cpp:
* Source/WebCore/page/csp/ContentSecurityPolicyMediaListDirective.h:
* Source/WebCore/page/csp/ContentSecurityPolicySourceListDirective.cpp:
* Source/WebCore/page/csp/ContentSecurityPolicySourceListDirective.h:
* Source/WebCore/page/csp/ContentSecurityPolicyTrustedTypesDirective.cpp:
* Source/WebCore/page/csp/ContentSecurityPolicyTrustedTypesDirective.h:
* Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp:
* Source/WebCore/page/scrolling/AsyncScrollingCoordinator.h:
* Source/WebCore/page/scrolling/ScrollingStateFixedNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStateFixedNode.h:
* Source/WebCore/page/scrolling/ScrollingStateFrameHostingNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStateFrameHostingNode.h:
* Source/WebCore/page/scrolling/ScrollingStateFrameScrollingNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStateFrameScrollingNode.h:
* Source/WebCore/page/scrolling/ScrollingStateOverflowScrollProxyNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStateOverflowScrollProxyNode.h:
* Source/WebCore/page/scrolling/ScrollingStatePluginHostingNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStatePluginHostingNode.h:
* Source/WebCore/page/scrolling/ScrollingStatePositionedNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStatePositionedNode.h:
* Source/WebCore/page/scrolling/ScrollingStateScrollingNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStateScrollingNode.h:
* Source/WebCore/page/scrolling/ScrollingStateStickyNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStateStickyNode.h:
* Source/WebCore/page/scrolling/ThreadedScrollingCoordinator.cpp:
* Source/WebCore/page/scrolling/ThreadedScrollingCoordinator.h:
* Source/WebCore/platform/DictationCaretAnimator.cpp:
* Source/WebCore/platform/DictationCaretAnimator.h:
* Source/WebCore/platform/MainThreadSharedTimer.cpp:
* Source/WebCore/platform/MainThreadSharedTimer.h:
* Source/WebCore/platform/OpacityCaretAnimator.cpp:
* Source/WebCore/platform/OpacityCaretAnimator.h:
* Source/WebCore/platform/SimpleCaretAnimator.cpp:
* Source/WebCore/platform/SimpleCaretAnimator.h:
* Source/WebCore/platform/audio/cocoa/AudioSessionCocoa.h:
* Source/WebCore/platform/audio/cocoa/AudioSessionCocoa.mm:
* Source/WebCore/platform/audio/ios/AudioSessionIOS.h:
* Source/WebCore/platform/audio/ios/AudioSessionIOS.mm:
* Source/WebCore/platform/audio/mac/AudioSessionMac.h:
* Source/WebCore/platform/audio/mac/AudioSessionMac.mm:
* Source/WebCore/platform/cf/KeyedDecoderCF.cpp:
* Source/WebCore/platform/cf/KeyedDecoderCF.h:
* Source/WebCore/platform/cf/KeyedEncoderCF.cpp:
* Source/WebCore/platform/cf/KeyedEncoderCF.h:
* Source/WebCore/platform/cocoa/NetworkExtensionContentFilter.h:
* Source/WebCore/platform/cocoa/NetworkExtensionContentFilter.mm:
* Source/WebCore/platform/cocoa/ParentalControlsContentFilter.h:
* Source/WebCore/platform/cocoa/ParentalControlsContentFilter.mm:
* Source/WebCore/platform/graphics/BifurcatedGraphicsContext.cpp:
* Source/WebCore/platform/graphics/BifurcatedGraphicsContext.h:
* Source/WebCore/platform/graphics/GraphicsLayer.cpp:
* Source/WebCore/platform/graphics/GraphicsLayer.h:
* Source/WebCore/platform/graphics/MediaPlayer.cpp:
* Source/WebCore/platform/graphics/NullGraphicsContext.h:
* Source/WebCore/platform/graphics/PathStream.cpp:
* Source/WebCore/platform/graphics/PathStream.h:
* Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateAVFoundationObjC.mm:
* Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaSourceAVFObjC.mm:
* Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaStreamAVFObjC.mm:
* Source/WebCore/platform/graphics/ca/GraphicsLayerCA.cpp:
* Source/WebCore/platform/graphics/ca/GraphicsLayerCA.h:
* Source/WebCore/platform/graphics/cg/GraphicsContextCG.cpp:
* Source/WebCore/platform/graphics/cg/GraphicsContextCG.h:
* Source/WebCore/platform/graphics/cg/PathCG.cpp:
* Source/WebCore/platform/graphics/cg/PathCG.h:
* Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.mm:
* Source/WebCore/platform/graphics/cocoa/controls/ControlFactoryCocoa.h:
* Source/WebCore/platform/graphics/cocoa/controls/ControlFactoryCocoa.mm:
* Source/WebCore/platform/graphics/controls/EmptyControlFactory.cpp:
* Source/WebCore/platform/graphics/controls/EmptyControlFactory.h:
* Source/WebCore/platform/graphics/ios/controls/ControlFactoryIOS.h:
* Source/WebCore/platform/graphics/ios/controls/ControlFactoryIOS.mm:
* Source/WebCore/platform/graphics/mac/controls/ControlFactoryMac.h:
* Source/WebCore/platform/graphics/mac/controls/ControlFactoryMac.mm:
* Source/WebCore/platform/mediastream/libwebrtc/LibWebRTCProviderCocoa.cpp:
* Source/WebCore/platform/mediastream/libwebrtc/LibWebRTCProviderCocoa.h:
* Source/WebCore/platform/mock/mediasource/MockMediaPlayerMediaSource.cpp:
* Source/WebCore/platform/network/cf/ResourceResponse.h:
* Source/WebCore/platform/network/cocoa/ResourceResponseCocoa.mm:
* Source/WebCore/platform/text/cocoa/LocaleCocoa.h:
* Source/WebCore/platform/text/cocoa/LocaleCocoa.mm:
* Source/WebCore/rendering/RenderSelectionGeometry.cpp:
* Source/WebCore/rendering/RenderSelectionGeometry.h:
* Source/WebCore/rendering/style/BasicShapes.cpp:
* Source/WebCore/rendering/style/BasicShapes.h:
* Source/WebCore/rendering/style/BasicShapesShape.cpp:
* Source/WebCore/rendering/style/BasicShapesShape.h:
* Source/WebCore/rendering/style/ContentData.cpp:
* Source/WebCore/rendering/style/ContentData.h:
* Source/WebCore/testing/MockContentFilter.cpp:
* Source/WebCore/testing/MockContentFilter.h:
* Source/WebCore/workers/WorkerRunLoop.cpp:
* Source/WebKit/GPUProcess/graphics/ImageBufferShareableAllocator.cpp:
* Source/WebKit/GPUProcess/graphics/ImageBufferShareableAllocator.h:
* Source/WebKit/GPUProcess/media/RemoteMediaResource.cpp:
* Source/WebKit/GPUProcess/media/RemoteMediaResource.h:
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWOriginStore.cpp:
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWOriginStore.h:
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWRegistrationStore.cpp:
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWRegistrationStore.h:
* Source/WebKit/NetworkProcess/storage/IDBStorageManager.cpp:
* Source/WebKit/NetworkProcess/storage/IDBStorageManager.h:
* Source/WebKit/NetworkProcess/storage/MemoryStorageArea.cpp:
* Source/WebKit/NetworkProcess/storage/MemoryStorageArea.h:
* Source/WebKit/NetworkProcess/storage/SQLiteStorageArea.cpp:
* Source/WebKit/NetworkProcess/storage/SQLiteStorageArea.h:
* Source/WebKit/NetworkProcess/storage/StorageAreaBase.h:
* Source/WebKit/Shared/RemoteLayerTree/DynamicContentScalingBifurcatedImageBuffer.h:
* Source/WebKit/Shared/RemoteLayerTree/DynamicContentScalingBifurcatedImageBuffer.mm:
* Source/WebKit/Shared/RemoteLayerTree/RemoteLayerWithInProcessRenderingBackingStore.h:
* Source/WebKit/Shared/RemoteLayerTree/RemoteLayerWithInProcessRenderingBackingStore.mm:
* Source/WebKit/Shared/RemoteLayerTree/RemoteLayerWithRemoteRenderingBackingStore.h:
* Source/WebKit/Shared/RemoteLayerTree/RemoteLayerWithRemoteRenderingBackingStore.mm:
* Source/WebKit/UIProcess/Cocoa/UIRemoteObjectRegistry.cpp:
* Source/WebKit/UIProcess/Cocoa/UIRemoteObjectRegistry.h:
* Source/WebKit/WebProcess/DigitalCredentials/DigitalCredentialsCoordinator.cpp:
* Source/WebKit/WebProcess/DigitalCredentials/DigitalCredentialsCoordinator.h:
* Source/WebKit/WebProcess/GPU/graphics/RemoteDisplayListRecorderProxy.cpp:
* Source/WebKit/WebProcess/GPU/graphics/RemoteDisplayListRecorderProxy.h:
* Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferProxy.cpp:
* Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferProxy.h:
* Source/WebKit/WebProcess/GPU/media/RemoteMediaPlayerManager.cpp:
* Source/WebKit/WebProcess/GPU/webrtc/MediaRecorderProvider.cpp:
* Source/WebKit/WebProcess/GPU/webrtc/MediaRecorderProvider.h:
* Source/WebKit/WebProcess/InjectedBundle/API/mac/WKWebProcessPlugInBrowserContextController.mm:
(-[WKWebProcessPlugInBrowserContextController _setFormDelegate:]):
(-[WKWebProcessPlugInBrowserContextController _setEditingDelegate:]):
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundleClient.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundleClient.h:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageContextMenuClient.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageContextMenuClient.h:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageEditorClient.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageEditorClient.h:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageFormClient.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageFormClient.h:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageLoaderClient.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageLoaderClient.h:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageResourceLoadClient.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageResourceLoadClient.h:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageUIClient.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageUIClient.h:
* Source/WebKit/WebProcess/Model/WebModelPlayerProvider.cpp:
* Source/WebKit/WebProcess/Model/WebModelPlayerProvider.h:
* Source/WebKit/WebProcess/Network/webrtc/LibWebRTCProvider.cpp:
* Source/WebKit/WebProcess/Network/webrtc/LibWebRTCProvider.h:
* Source/WebKit/WebProcess/WebAuthentication/WebAuthenticatorCoordinator.cpp:
(WebKit::WebCore::isWebBrowser): Deleted.
* Source/WebKit/WebProcess/WebAuthentication/WebAuthenticatorCoordinator.h:
* Source/WebKit/WebProcess/WebCoreSupport/WebPerformanceLoggingClient.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/WebPerformanceLoggingClient.h:
* Source/WebKit/WebProcess/WebCoreSupport/WebSpeechRecognitionProvider.h:
* Source/WebKit/WebProcess/WebCoreSupport/WebValidationMessageClient.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/WebValidationMessageClient.h:
* Source/WebKit/WebProcess/WebCoreSupport/WebWorkerClient.cpp:
* Source/WebKit/WebProcess/WebPage/Cocoa/WebRemoteObjectRegistry.cpp:
* Source/WebKit/WebProcess/WebPage/Cocoa/WebRemoteObjectRegistry.h:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/GraphicsLayerCARemote.h:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/GraphicsLayerCARemote.mm:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteLayerTreeDrawingArea.h:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteLayerTreeDrawingArea.mm:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollbarsController.h:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollbarsController.mm:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollingCoordinator.h:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollingCoordinator.mm:
* Source/WebKit/WebProcess/WebPage/mac/RemoteLayerTreeDrawingAreaMac.h:
* Source/WebKit/WebProcess/WebPage/mac/RemoteLayerTreeDrawingAreaMac.mm:
* Source/WebKit/WebProcess/WebPage/mac/TiledCoreAnimationDrawingArea.h:
* Source/WebKit/WebProcess/WebPage/mac/TiledCoreAnimationDrawingArea.mm:
* Source/WebKitLegacy/mac/WebCoreSupport/WebValidationMessageClient.h:
* Source/WebKitLegacy/mac/WebCoreSupport/WebValidationMessageClient.mm:

Canonical link: <a href="https://commits.webkit.org/283286@main">https://commits.webkit.org/283286@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/c5b8331268542e602dc0a70115ef7175847a9777

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/65836 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/45209 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/18455 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/69865 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/16445 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/67954 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/53008 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/16727 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/52851 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/11427 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/68903 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/41732 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/56988 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/41/builds/33483 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/38406 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/14367 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/15321 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/60250 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/14711 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/71568 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/9791 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/62/builds/14130 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/60159 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/9823 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/57055 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/60445 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/14509 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/8088 "Passed tests") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/119/builds/1732 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/44/builds/41017 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/42093 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/43276 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/41837 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

MenloDorian commented on Source/WebCore/Modules/model-element/dummy/DummyModelPlayerProvider.h:
```diff
@@ -27,9 +27,12 @@
 
 #include "ModelPlayerProvider.h"
 
+#include <wtf/TZoneMallocInlines.h>
```
Does this really need to be `TZoneMallocInlines.h`?  You have a .cpp file, and this is a Dummy class.  Why does it need the inlining?

MenloDorian replied:
nit: please remove the empty line above.

msaboff replied:
Empty line removed.

Changed include to TZoneMalloc.h.

MenloDorian commented on Source/WebCore/crypto/parameters/CryptoAlgorithmAesCbcCfbParams.h:
```diff
@@ -27,11 +27,13 @@
 namespace WebCore {
 
 class CryptoAlgorithmAesCbcCfbParams final : public CryptoAlgorithmParameters {
+    WTF_MAKE_TZONE_ALLOCATED_INLINE(CryptoAlgorithmAesCbcCfbParams);
```
Does this really need to be inlined?  We should revisit this later and make sure we're not inlining unnecessarily for all these allocators.

msaboff replied:
I changed this instance to TZoneMalloc.h.

MenloDorian commented on Source/WebCore/page/DummySpeechRecognitionProvider.h:
```diff
@@ -28,9 +28,12 @@
 namespace WebCore {
 
 class DummySpeechRecognitionProvider final : public SpeechRecognitionProvider {
+    WTF_MAKE_TZONE_ALLOCATED_INLINE(DummySpeechRecognitionProvider);
```
Ditto.  Does this really need to be inlined?

msaboff replied:
There are two places where `WTF_MAKE_TZONE_ALLOCATED_INLINE` is appropriate.
1. Classes defined in a `.cpp/.mm` file.
2. For a class defined in a `.h` file that doesn't have a corresponding `.cpp/.mm` file.

MenloDorian commented on Source/WebCore/platform/cocoa/ParentalControlsContentFilter.h:
```diff
@@ -28,6 +28,7 @@
 #include "PlatformContentFilter.h"
 #include <wtf/Compiler.h>
 #include <wtf/RetainPtr.h>
+#include <wtf/TZoneMallocInlines.h>
```
Do we need the Inlines.h header here?

msaboff replied:
I changed this instance to TZoneMalloc.h.

MenloDorian commented on Source/WebKit/WebProcess/GPU/webrtc/MediaRecorderProvider.h:
```diff
@@ -27,11 +27,14 @@
 
 #include <WebCore/MediaRecorderProvider.h>
 
+#include <wtf/TZoneMalloc.h>
```
nit: Remove the empty line above.

msaboff replied:
Done.

MenloDorian commented on Source/WebKit/WebProcess/WebPage/mac/TiledCoreAnimationDrawingArea.h:
```diff
@@ -35,6 +35,7 @@
 #include <wtf/HashMap.h>
 #include <wtf/RetainPtr.h>
 #include <wtf/RunLoop.h>
+#include <wtf/TZoneMallocInlines.h>
```
Does this need to be Inlines.h?

msaboff replied:
Changed it to TZoneMalloc.h.

MenloDorian commented on Source/WebKit/Shared/RemoteLayerTree/RemoteLayerWithRemoteRenderingBackingStore.h:
```diff
@@ -27,6 +27,8 @@
 
 #include "RemoteLayerBackingStore.h"
 
+#include <wtf/TZoneMalloc.h>
```
nit: please remove the empty line above.

MenloDorian commented on Source/WebCore/platform/audio/ios/AudioSessionIOS.h:
```diff
@@ -29,6 +29,8 @@
 
 #include "AudioSessionCocoa.h"
 
+#include <wtf/TZoneMalloc.h>
```
nit: Remove the empty line above.

msaboff replied:
Done.

MenloDorian replied:
nit: empty line is still present above.

MenloDorian commented on Source/WebCore/page/DummySpeechRecognitionProvider.h:
```diff
@@ -28,9 +28,12 @@
 
 #include "SpeechRecognitionProvider.h"
 
+#include <wtf/TZoneMallocInlines.h>
```
nit: Remove the empty line above.

msaboff replied:
Done.

MenloDorian commented on Source/WebCore/loader/EmptyClients.cpp:
```diff
@@ -1163,6 +1163,7 @@ class EmptyStorageSessionProvider final : public StorageSessionProvider {
 };
 
 class EmptyMediaRecorderProvider final : public MediaRecorderProvider {
+    WTF_MAKE_TZONE_ALLOCATED_INLINE(MediaRecorderProvider);
```
Why use the TZone for `MediaRecorderProvider` and not one for `EmptyMediaRecorderProvider`?

msaboff replied:
Typo.  Changed to `EmptyMediaRecorderProvider?`.

MenloDorian reviewed (approved):
r=me with 1 nit left to fix.

msaboff committed:
[TZone] Add annotations to unannotated subclasses
https://bugs.webkit.org/show_bug.cgi?id=279191
rdar://135344436

Reviewed by Mark Lam.

Added TZone allocation annotation to unannotated subclasses that inherit from annotated base classes.

* Source/WebCore/Modules/model-element/dummy/DummyModelPlayerProvider.cpp:
* Source/WebCore/Modules/model-element/dummy/DummyModelPlayerProvider.h:
* Source/WebCore/PAL/pal/text/TextCodecCJK.cpp:
* Source/WebCore/PAL/pal/text/TextCodecCJK.h:
* Source/WebCore/PAL/pal/text/TextCodecICU.cpp:
* Source/WebCore/PAL/pal/text/TextCodecICU.h:
* Source/WebCore/PAL/pal/text/TextCodecReplacement.cpp:
* Source/WebCore/PAL/pal/text/TextCodecReplacement.h:
* Source/WebCore/PAL/pal/text/TextCodecSingleByte.cpp:
* Source/WebCore/PAL/pal/text/TextCodecSingleByte.h:
* Source/WebCore/PAL/pal/text/TextCodecUTF16.cpp:
* Source/WebCore/PAL/pal/text/TextCodecUTF16.h:
* Source/WebCore/PAL/pal/text/TextCodecUTF8.cpp:
* Source/WebCore/PAL/pal/text/TextCodecUTF8.h:
* Source/WebCore/PAL/pal/text/TextCodecUserDefined.cpp:
* Source/WebCore/PAL/pal/text/TextCodecUserDefined.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmAesCbcCfbParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmAesCtrParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmAesGcmParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmAesKeyParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmEcKeyParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmEcdhKeyDeriveParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmEcdsaParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmHkdfParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmHmacKeyParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmPbkdf2Params.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmRsaHashedImportParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmRsaKeyGenParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmRsaOaepParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmRsaPssParams.h:
* Source/WebCore/crypto/parameters/CryptoAlgorithmX25519Params.h:
* Source/WebCore/dom/ActiveDOMObject.cpp:
* Source/WebCore/dom/EventLoop.cpp:
* Source/WebCore/editing/Editor.h:
* Source/WebCore/html/BaseCheckableInputType.cpp:
* Source/WebCore/html/BaseCheckableInputType.h:
* Source/WebCore/html/BaseClickableWithKeyInputType.cpp:
* Source/WebCore/html/BaseClickableWithKeyInputType.h:
* Source/WebCore/html/BaseDateAndTimeInputType.cpp:
* Source/WebCore/html/BaseDateAndTimeInputType.h:
* Source/WebCore/html/CheckboxInputType.cpp:
* Source/WebCore/html/CheckboxInputType.h:
* Source/WebCore/html/HTMLMediaElement.cpp:
* Source/WebCore/html/HiddenInputType.cpp:
* Source/WebCore/html/HiddenInputType.h:
* Source/WebCore/html/RadioInputType.cpp:
* Source/WebCore/html/RadioInputType.h:
* Source/WebCore/html/RangeInputType.cpp:
* Source/WebCore/html/RangeInputType.h:
* Source/WebCore/html/TextFieldInputType.cpp:
* Source/WebCore/html/TextFieldInputType.h:
* Source/WebCore/loader/EmptyClients.cpp:
* Source/WebCore/loader/MediaResourceLoader.cpp:
* Source/WebCore/loader/MediaResourceLoader.h:
* Source/WebCore/page/Chrome.cpp:
* Source/WebCore/page/Chrome.h:
* Source/WebCore/page/DummySpeechRecognitionProvider.h:
* Source/WebCore/page/csp/ContentSecurityPolicyMediaListDirective.cpp:
* Source/WebCore/page/csp/ContentSecurityPolicyMediaListDirective.h:
* Source/WebCore/page/csp/ContentSecurityPolicySourceListDirective.cpp:
* Source/WebCore/page/csp/ContentSecurityPolicySourceListDirective.h:
* Source/WebCore/page/csp/ContentSecurityPolicyTrustedTypesDirective.cpp:
* Source/WebCore/page/csp/ContentSecurityPolicyTrustedTypesDirective.h:
* Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp:
* Source/WebCore/page/scrolling/AsyncScrollingCoordinator.h:
* Source/WebCore/page/scrolling/ScrollingStateFixedNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStateFixedNode.h:
* Source/WebCore/page/scrolling/ScrollingStateFrameHostingNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStateFrameHostingNode.h:
* Source/WebCore/page/scrolling/ScrollingStateFrameScrollingNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStateFrameScrollingNode.h:
* Source/WebCore/page/scrolling/ScrollingStateOverflowScrollProxyNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStateOverflowScrollProxyNode.h:
* Source/WebCore/page/scrolling/ScrollingStatePluginHostingNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStatePluginHostingNode.h:
* Source/WebCore/page/scrolling/ScrollingStatePositionedNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStatePositionedNode.h:
* Source/WebCore/page/scrolling/ScrollingStateScrollingNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStateScrollingNode.h:
* Source/WebCore/page/scrolling/ScrollingStateStickyNode.cpp:
* Source/WebCore/page/scrolling/ScrollingStateStickyNode.h:
* Source/WebCore/page/scrolling/ThreadedScrollingCoordinator.cpp:
* Source/WebCore/page/scrolling/ThreadedScrollingCoordinator.h:
* Source/WebCore/platform/DictationCaretAnimator.cpp:
* Source/WebCore/platform/DictationCaretAnimator.h:
* Source/WebCore/platform/MainThreadSharedTimer.cpp:
* Source/WebCore/platform/MainThreadSharedTimer.h:
* Source/WebCore/platform/OpacityCaretAnimator.cpp:
* Source/WebCore/platform/OpacityCaretAnimator.h:
* Source/WebCore/platform/SimpleCaretAnimator.cpp:
* Source/WebCore/platform/SimpleCaretAnimator.h:
* Source/WebCore/platform/audio/cocoa/AudioSessionCocoa.h:
* Source/WebCore/platform/audio/cocoa/AudioSessionCocoa.mm:
* Source/WebCore/platform/audio/ios/AudioSessionIOS.h:
* Source/WebCore/platform/audio/ios/AudioSessionIOS.mm:
* Source/WebCore/platform/audio/mac/AudioSessionMac.h:
* Source/WebCore/platform/audio/mac/AudioSessionMac.mm:
* Source/WebCore/platform/cf/KeyedDecoderCF.cpp:
* Source/WebCore/platform/cf/KeyedDecoderCF.h:
* Source/WebCore/platform/cf/KeyedEncoderCF.cpp:
* Source/WebCore/platform/cf/KeyedEncoderCF.h:
* Source/WebCore/platform/cocoa/NetworkExtensionContentFilter.h:
* Source/WebCore/platform/cocoa/NetworkExtensionContentFilter.mm:
* Source/WebCore/platform/cocoa/ParentalControlsContentFilter.h:
* Source/WebCore/platform/cocoa/ParentalControlsContentFilter.mm:
* Source/WebCore/platform/graphics/BifurcatedGraphicsContext.cpp:
* Source/WebCore/platform/graphics/BifurcatedGraphicsContext.h:
* Source/WebCore/platform/graphics/GraphicsLayer.cpp:
* Source/WebCore/platform/graphics/GraphicsLayer.h:
* Source/WebCore/platform/graphics/MediaPlayer.cpp:
* Source/WebCore/platform/graphics/NullGraphicsContext.h:
* Source/WebCore/platform/graphics/PathStream.cpp:
* Source/WebCore/platform/graphics/PathStream.h:
* Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateAVFoundationObjC.mm:
* Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaSourceAVFObjC.mm:
* Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaStreamAVFObjC.mm:
* Source/WebCore/platform/graphics/ca/GraphicsLayerCA.cpp:
* Source/WebCore/platform/graphics/ca/GraphicsLayerCA.h:
* Source/WebCore/platform/graphics/cg/GraphicsContextCG.cpp:
* Source/WebCore/platform/graphics/cg/GraphicsContextCG.h:
* Source/WebCore/platform/graphics/cg/PathCG.cpp:
* Source/WebCore/platform/graphics/cg/PathCG.h:
* Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.mm:
* Source/WebCore/platform/graphics/cocoa/controls/ControlFactoryCocoa.h:
* Source/WebCore/platform/graphics/cocoa/controls/ControlFactoryCocoa.mm:
* Source/WebCore/platform/graphics/controls/EmptyControlFactory.cpp:
* Source/WebCore/platform/graphics/controls/EmptyControlFactory.h:
* Source/WebCore/platform/graphics/ios/controls/ControlFactoryIOS.h:
* Source/WebCore/platform/graphics/ios/controls/ControlFactoryIOS.mm:
* Source/WebCore/platform/graphics/mac/controls/ControlFactoryMac.h:
* Source/WebCore/platform/graphics/mac/controls/ControlFactoryMac.mm:
* Source/WebCore/platform/mediastream/libwebrtc/LibWebRTCProviderCocoa.cpp:
* Source/WebCore/platform/mediastream/libwebrtc/LibWebRTCProviderCocoa.h:
* Source/WebCore/platform/mock/mediasource/MockMediaPlayerMediaSource.cpp:
* Source/WebCore/platform/network/cf/ResourceResponse.h:
* Source/WebCore/platform/network/cocoa/ResourceResponseCocoa.mm:
* Source/WebCore/platform/text/cocoa/LocaleCocoa.h:
* Source/WebCore/platform/text/cocoa/LocaleCocoa.mm:
* Source/WebCore/rendering/RenderSelectionGeometry.cpp:
* Source/WebCore/rendering/RenderSelectionGeometry.h:
* Source/WebCore/rendering/style/BasicShapes.cpp:
* Source/WebCore/rendering/style/BasicShapes.h:
* Source/WebCore/rendering/style/BasicShapesShape.cpp:
* Source/WebCore/rendering/style/BasicShapesShape.h:
* Source/WebCore/rendering/style/ContentData.cpp:
* Source/WebCore/rendering/style/ContentData.h:
* Source/WebCore/testing/MockContentFilter.cpp:
* Source/WebCore/testing/MockContentFilter.h:
* Source/WebCore/workers/WorkerRunLoop.cpp:
* Source/WebKit/GPUProcess/graphics/ImageBufferShareableAllocator.cpp:
* Source/WebKit/GPUProcess/graphics/ImageBufferShareableAllocator.h:
* Source/WebKit/GPUProcess/media/RemoteMediaResource.cpp:
* Source/WebKit/GPUProcess/media/RemoteMediaResource.h:
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWOriginStore.cpp:
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWOriginStore.h:
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWRegistrationStore.cpp:
* Source/WebKit/NetworkProcess/ServiceWorker/WebSWRegistrationStore.h:
* Source/WebKit/NetworkProcess/storage/IDBStorageManager.cpp:
* Source/WebKit/NetworkProcess/storage/IDBStorageManager.h:
* Source/WebKit/NetworkProcess/storage/MemoryStorageArea.cpp:
* Source/WebKit/NetworkProcess/storage/MemoryStorageArea.h:
* Source/WebKit/NetworkProcess/storage/SQLiteStorageArea.cpp:
* Source/WebKit/NetworkProcess/storage/SQLiteStorageArea.h:
* Source/WebKit/NetworkProcess/storage/StorageAreaBase.h:
* Source/WebKit/Shared/RemoteLayerTree/DynamicContentScalingBifurcatedImageBuffer.h:
* Source/WebKit/Shared/RemoteLayerTree/DynamicContentScalingBifurcatedImageBuffer.mm:
* Source/WebKit/Shared/RemoteLayerTree/RemoteLayerWithInProcessRenderingBackingStore.h:
* Source/WebKit/Shared/RemoteLayerTree/RemoteLayerWithInProcessRenderingBackingStore.mm:
* Source/WebKit/Shared/RemoteLayerTree/RemoteLayerWithRemoteRenderingBackingStore.h:
* Source/WebKit/Shared/RemoteLayerTree/RemoteLayerWithRemoteRenderingBackingStore.mm:
* Source/WebKit/UIProcess/Cocoa/UIRemoteObjectRegistry.cpp:
* Source/WebKit/UIProcess/Cocoa/UIRemoteObjectRegistry.h:
* Source/WebKit/WebProcess/DigitalCredentials/DigitalCredentialsCoordinator.cpp:
* Source/WebKit/WebProcess/DigitalCredentials/DigitalCredentialsCoordinator.h:
* Source/WebKit/WebProcess/GPU/graphics/RemoteDisplayListRecorderProxy.cpp:
* Source/WebKit/WebProcess/GPU/graphics/RemoteDisplayListRecorderProxy.h:
* Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferProxy.cpp:
* Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferProxy.h:
* Source/WebKit/WebProcess/GPU/media/RemoteMediaPlayerManager.cpp:
* Source/WebKit/WebProcess/GPU/webrtc/MediaRecorderProvider.cpp:
* Source/WebKit/WebProcess/GPU/webrtc/MediaRecorderProvider.h:
* Source/WebKit/WebProcess/InjectedBundle/API/mac/WKWebProcessPlugInBrowserContextController.mm:
(-[WKWebProcessPlugInBrowserContextController _setFormDelegate:]):
(-[WKWebProcessPlugInBrowserContextController _setEditingDelegate:]):
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundleClient.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundleClient.h:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageContextMenuClient.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageContextMenuClient.h:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageEditorClient.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageEditorClient.h:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageFormClient.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageFormClient.h:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageLoaderClient.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageLoaderClient.h:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageResourceLoadClient.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageResourceLoadClient.h:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageUIClient.cpp:
* Source/WebKit/WebProcess/InjectedBundle/InjectedBundlePageUIClient.h:
* Source/WebKit/WebProcess/Model/WebModelPlayerProvider.cpp:
* Source/WebKit/WebProcess/Model/WebModelPlayerProvider.h:
* Source/WebKit/WebProcess/Network/webrtc/LibWebRTCProvider.cpp:
* Source/WebKit/WebProcess/Network/webrtc/LibWebRTCProvider.h:
* Source/WebKit/WebProcess/WebAuthentication/WebAuthenticatorCoordinator.cpp:
(WebKit::WebCore::isWebBrowser): Deleted.
* Source/WebKit/WebProcess/WebAuthentication/WebAuthenticatorCoordinator.h:
* Source/WebKit/WebProcess/WebCoreSupport/WebPerformanceLoggingClient.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/WebPerformanceLoggingClient.h:
* Source/WebKit/WebProcess/WebCoreSupport/WebSpeechRecognitionProvider.h:
* Source/WebKit/WebProcess/WebCoreSupport/WebValidationMessageClient.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/WebValidationMessageClient.h:
* Source/WebKit/WebProcess/WebCoreSupport/WebWorkerClient.cpp:
* Source/WebKit/WebProcess/WebPage/Cocoa/WebRemoteObjectRegistry.cpp:
* Source/WebKit/WebProcess/WebPage/Cocoa/WebRemoteObjectRegistry.h:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/GraphicsLayerCARemote.h:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/GraphicsLayerCARemote.mm:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteLayerTreeDrawingArea.h:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteLayerTreeDrawingArea.mm:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollbarsController.h:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollbarsController.mm:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollingCoordinator.h:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollingCoordinator.mm:
* Source/WebKit/WebProcess/WebPage/mac/RemoteLayerTreeDrawingAreaMac.h:
* Source/WebKit/WebProcess/WebPage/mac/RemoteLayerTreeDrawingAreaMac.mm:
* Source/WebKit/WebProcess/WebPage/mac/TiledCoreAnimationDrawingArea.h:
* Source/WebKit/WebProcess/WebPage/mac/TiledCoreAnimationDrawingArea.mm:
* Source/WebKitLegacy/mac/WebCoreSupport/WebValidationMessageClient.h:
* Source/WebKitLegacy/mac/WebCoreSupport/WebValidationMessageClient.mm:

Canonical link: https://commits.webkit.org/283286@main
